src/main/java/edu/umich/lib/dor/replicaexperiment/service/DepositDirectory.java

```diff
@@ -0,0 +1,45 @@
+package edu.umich.lib.dor.replicaexperiment.service;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+public class DepositDirectory {
+    private static final Log log = LogFactory.getLog(OcflFilesystemRepositoryClient.class);
```

Comment:
Kinda feel like this could be made more abstract as a `Storage` or `StorageClient` class. Thoughts?

Reply:
See https://github.com/ssciolla/dor-replica-experiment/pull/13/commits/4952477b0f4770ecfd5972cf7c73f346a68aa86d for refactor related to this.